.github/workflows/build.yml

```
@@ -29,10 +29,10 @@ jobs:
   # With this we catch potential issues already in the PR.
   ios-swift-release:
     name: Release Build of iOS Swift
-    runs-on: macos-14
+    runs-on: macos15
```

`m`: I think this should be
```suggestion
    runs-on: macos-15
```

---

Thanks!